### PR TITLE
SRU: add MARC21 physical description & ISBN

### PR DESCRIPTION
**Pull request by rerowep**
* Adds JSON to MARC21 physical description transformation.
 * Adds JSON to MARC21 ISBN transformation.
    
## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/sru/explaine.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2020 RERO
+# Copyright (C) 2020 UCLouvain
```
Comment (iGormilhit) [inline rero_ils/modules/sru/explaine.py:5]:
```suggestion
# Copyright (C) 2021 RERO
# Copyright (C) 2021 UCLouvain
```

File: rero_ils/modules/sru/views.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
```
Comment (iGormilhit) [inline rero_ils/modules/sru/views.py:4]:
```suggestion
# Copyright (C) 2021 RERO
```

File: tests/api/sru/test_sru_rest.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
```
Comment (iGormilhit) [inline tests/api/sru/test_sru_rest.py:4]:
```suggestion
# Copyright (C) 2021 RERO
```

File: tests/unit/test_cql_parser.py [hunk 4]
```diff
@@ -0,0 +1,365 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
```
Comment (iGormilhit) [inline tests/unit/test_cql_parser.py:4]:
```suggestion
# Copyright (C) 2021 RERO
```

File: tests/unit/test_sru_explain.py [hunk 5]
```diff
@@ -0,0 +1,263 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
```
Comment (iGormilhit) [inline tests/unit/test_sru_explain.py:4]:
```suggestion
# Copyright (C) 2021 RERO
```

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit title: SRU capitalized, do not forget a verb.

Some comments are on the previous commit, sorry. Should be corrected on https://github.com/rero/rero-ils/pull/1834.

File: rero_ils/modules/documents/dojson/contrib/jsontomarc21/__init__.py [hunk 6]
```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""RERO MARC21 to JSON."""
+"""JSON to MARC21."""
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/documents/dojson/contrib/jsontomarc21/__init__.py]: Generic

[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/documents/dojson/contrib/jsontomarc21/__init__.py]: Generic

**Review by zannkukai — APPROVED** [review]

**Pull request merged**